app/webpacker/components/ContactEditProfilePage/index.jsx

```diff
@@ -45,6 +36,39 @@ export default function ContactEditProfilePage({ loggedInUserId, recaptchaPublic
     );
   }
 
+  if (queryParams.editOthersProfile) {
+    return (
+      <ContactEditProfileOthers
+        setContactSuccess={setContactSuccess}
+        recaptchaPublicKey={recaptchaPublicKey}
+      />
+    );
+  }
+  return (
+    <ContactEditProfileSelf
+      setContactSuccess={setContactSuccess}
+      recaptchaPublicKey={recaptchaPublicKey}
+    />
+  );
```

Comment:
I'm not a big fan of this duplication approach. It should definitely be possible to have just one `ContactEditProfile` component, and give it a WCA ID search input which either happens to be pre-filled (and disabled, in case of self-edit) or enabled and open for the user to search whathever ID they want to edit.

Reply:
Okay, I merged both of them. I split them because I feel they were bit complicated when together.